sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -159,7 +159,7 @@ object Size {
        4
   """,
   since = "3.3.0",
-  group = "collection_funcs")
+  group = "array_funcs")
```

Comment:
this too

Reply:
in python, there are many functions in the incorrect groups

I will resolve it in a separate PR

Reply:
🆗